Update Mayflower docs homepage

clairesunstudio opened the pull request.

legostud commented:
I'm confused by this change.  Not having a number seems to defeat the purpose of footnotes since we don't know what this relates to which makes me wonder if the request was actually to remove the link from number instead of removing the number.

Outside of that there are a couple issues here.  
1. The `js-footnote` class needs to be removed to prevent the JavaScript from being processed on this note.
2. There is extra padding on the left side of the footnote where the number used to be.  Should that still be there when the number is removed?

legostud commented on styleguide/source/_patterns/02-molecules/footnote.twig:
```diff
@@ -1,5 +1,10 @@
-<div class="ma__footnote js-footnote" tabindex="-1">
-  {% if footnote.number %}<a class="ma__footnote__link js-footnote-link" id="{{ footnote.id }}" href="{{ footnote.target }}">{{ footnote.number }}</a>{% endif %}
+{# Option to remove js functionality and hide the footnote link (this is an implementation workaround). #}
+{% set footnoteJs = footnote.number ? '' : 'js-footnote' %}
```
this is backwards.

jesconstantine replied:
Ha - I was just testing YOU. :) I'll fix it, thanks.

jesconstantine replied:
Those tricky ternaries.  Fixed in https://github.com/massgov/mayflower/pull/541/commits/812aa25546acd5b6cebc3e9f3509e41f99a389ec

clairesunstudio committed:
initial commit

clairesunstudio committed:
edit

clairesunstudio committed:
add calcualtors

clairesunstudio committed:
create community project page

clairesunstudio committed:
move into docs

clairesunstudio committed:
add to menu

juliagutierrez committed:
Added stuff

Fixed a row with MassDEP info, added the DTC form release search application, and added the EEC salesforce application. 

@avrilpearl @clairesunstudio , can you confirm that we do want the EEC in there and that I wrote the right things for the different columns for that row?

juliagutierrez committed:
community edits

changed things in the table, removed the community projects table, added a join our community section, updated the submit your project bullets, tried to reference headings in getting started section.

clairesunstudio commented on docs/README.md:
```diff
@@ -5,29 +5,45 @@
-* The most recent version of the Mayflower Design System that the project targets
+* The code repository if it is in Github or equivalent (if public)
+* What type of Mayflower implementation you are using
+* What Mayflower Desyn System version you are using 
```
```suggestion
* What Mayflower Design System version you are using 
```

clairesunstudio committed:
Update docs/README.md
